client/my-sites/customer-home/cards/tasks/site-setup-list/current-task-item.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { Button } from '@automattic/components';
+import Badge from 'components/badge';
```

Comment:
Actually I've been confused about this thing since the beginning. At most places I've seen that there is that comment which felt wrong to me especially this one: https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx
So I decided go with this one without altering it.